src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.data.person;
+
+
+public class Contact{
+

```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.